src/hrc1155.spec.ts

```diff
@@ -33,178 +33,313 @@ describe('HRC1155 Contract Interface', () => {
     expect(contract).to.not.be.undefined
   })
 
-  describe.skip('balanceOf', () => {
+  describe('balanceOf', () => {
     it('should get the number of tokens in the specified account with id as a number', async () => {
-      const balance = await contract.balanceOf(TEST_ADDRESS_1, TOKEN_GOLD)
+      const mockedBalance = await FAKE_BALANCE_HRC1155
+      const stub = sinon.stub(contract, 'call').withArgs('balanceOf', [TEST_ADDRESS_1, TOKEN_GOLD], TX_OPTIONS)
+      stub.resolves().returns(FAKE_BALANCE_HRC1155)
 
-      expect(balance).to.not.be.null
-      expect(balance).to.not.be.undefined
-      expect(balance).to.be.an.instanceof(BN)
-      expect(balance.gt(new BN(0))).to.be.true
+      const call = await contract.balanceOf(TEST_ADDRESS_1, TOKEN_GOLD, TX_OPTIONS)
```

Comment:
nit: I'd call this `result` or `balance` so that it's clearer. Similar in other tests

Reply:
agree!

Reply:
Okay!